pkg/helper/client.go

```diff
@@ -102,5 +113,17 @@ func ApplyDesiredState(nodeNetworkState *nmstatev1alpha1.NodeNetworkState) (stri
 	if len(desiredState) == 0 {
 		return "Ignoring empty desired state", nil
 	}
-	return set(string(nodeNetworkState.Spec.DesiredState))
+	outputSet, err := set(string(nodeNetworkState.Spec.DesiredState))
+	if err != nil {
+		return outputSet, err
+	}
+	bridgesUp, err := getBridgesUp(nodeNetworkState.Spec.DesiredState)
+	if err != nil {
+		return "", fmt.Errorf("error retrieving up bridges from desired state: %v", err)
```

Comment:
This and the next return are questionable. We have no way to rollback if our hack after `nmstatectl set` fails. How would we report it in conditions? Which state are we in? I think we should at least log a big warning.

Reply:
We need to run this after nmstate since, we create the bridges there, the proper option would be to use commit/rollback and do the rollback in case vlan-filtering fails, so we consider the two operatoins as one.

For now I am going to add a big warning

Is that ok ?

Reply:
Yes, big warning is fine. And a TODO to handle the rollback once we have needed tools in `nmstatectl`.

Reply:
Well as commented it's going to re-concile if vlan-filtering fails so that's like a big warning, so added the TODO about commit/rollback